### PR TITLE
Global chat - desktop

### PR DESCRIPTION
**Pull request by adrake33**
Desktop portion of #4112. (Mobile portion is still in progress and hidden for now.) The Orbit server is not set up yet, so the visual part of global chat is hard-coded for now.

### CODE REVIEW

File: packages/augur-ui/src/modules/common/global-chat.styles.less [hunk 1]
```diff
@@ -0,0 +1,103 @@
+@import (reference) '~assets/styles/shared';
+
+.GlobalChat {
+  background: @color-dark-grey;
+  bottom: @size-4;
+  position: fixed;
+  right: @size-22;
+  z-index: 2;
```
Comment (phoebemirman) [inline packages/augur-ui/src/modules/common/global-chat.styles.less]: z indeex has constants in misc_redesign

File: packages/augur-ui/src/modules/common/global-chat.styles.less [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+      transform: rotate(180deg);
+      width: @size-12;
+    }
+  }
+
+  > div {
+    border: 1px solid #0E0E0F;
+
+    // Header section
+    > div:first-of-type {
```
Comment (phoebemirman) [inline packages/augur-ui/src/modules/common/global-chat.styles.less]: can you use color constant too

Comment (adrake33) [reply packages/augur-ui/src/modules/common/global-chat.styles.less]: Oops, yeah I meant to go back and fix that before creating my PR.

**Review by phoebemirman — APPROVED** [review]

File: packages/augur-ui/src/modules/common/global-chat.tsx [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+export interface GlobalChatProps {
+  toggleGlobalChat: Function;
+}
+
+export interface GlobalChatState {
+  isOpen: boolean;
+}
+
+export class GlobalChat extends React.Component<GlobalChatProps, GlobalChatState> {
```
Comment (bthaile) [inline packages/augur-ui/src/modules/common/global-chat.tsx]: make this a functional component

use `useState`

**Review by bthaile — CHANGES_REQUESTED** [review]
convert to functional component

**Review by bthaile — APPROVED** [review]
lgtm, cleans up that component

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 🍻

 This link will always point to this PR's latest changes: https://peek.run/7g6hvx0

**Pull request merged**